docs/content/docs/getting-started/generate-providers.md

```diff
@@ -32,6 +33,10 @@ to reduce the generation time.
 make terraform VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google" PRODUCT=foo
```

Comment:
if we're standardizing on `make provider` then we should replace all references to `make terraform` in our docs with `make provider` instead. We should update the container-based process as well so that people don't get confused about the discrepancy if they look in the Dockerfile.